src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  width: 200px;
+  background-color: $white;
+  border: 1px solid $grey;
+  border-radius: 5px;
+
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+    margin: 0;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin: 4px 0 16px;
+  }
```

Comment:
@RomeoKs i kind of get it it's easier to define top & bottom margin for just one element and remove margins for the previous one
BUT
imagine you'll need to reorder the contents of a card - you get the idea of what happens)
<img width="826" alt="Monosnap Catalog 2022-11-09 11-42-12" src="https://user-images.githubusercontent.com/39308717/200796223-de2af1c8-8451-418b-ab60-e50de9aa7815.png">

so still better to stick with either top or bottom margin and use for all the children of a card.
Let's apply this)

Reply:
Just a few days ago I understood the concept.
Not only it makes sense, but I remembered issued that I had in the past and could not explain myself why.